docs/custom-flows/oauth-connections.mdx

```diff
@@ -5,173 +5,172 @@ description: Learn how to leverage the Clerk SDK to build completely custom OAut
-
-<Images
-  width={3024}
-  height={1654}
-  src="/docs/images/custom-flows/mobile-oauth-allowlist.jpg"
-  alt="The 'Social Connections' page of the Clerk Dashboard, with the 'Settings' tab selected. The 'Settings' tab shows a section titled 'allowlist for mobile OAuth redirect' and has an input for redirect URLs."
-/>
-
-## OAuth account transfer flow
-
-When a user initiates an OAuth verification during sign-in or sign-up, it may sometimes be necessary to move the verification between the two flows.
-
-For example, if someone already has an account, and tries to go through the sign up flow with the same OAuth account, they can't perform a successful sign-up again. Or, if someone attempts to sign in with their OAuth credentials but does not yet have an account, they won't be signed in to the account. For these scenarios, Clerk provides "account transfers."
-
-<CodeBlockTabs type="framework" options={["JavaScript"]}>
-```js
-// Get sign-in and sign-up objects in the OAuth callback page
-const { signIn, signUp } = window.Clerk.client;
-
-// If the user has an account in your application, but does not yet 
-// have an oauth account connected, you can use the transfer method to 
-// forward that information.
-
-const userExistsButNeedsToSignIn =
-  signUp.verifications.externalAccount.status === "transferable" &&
-  signUp.verifications.externalAccount.error?.code ===
-    "external_account_exists";
-
-if (userExistsButNeedsToSignIn) {
-  const res = await signIn.create({ transfer: true });
-  if (res.status === "complete") {
-    window.Clerk.setActive({
-      session: res.createdSessionId,
-    });
-  }
-}
-
-// If the user has an existing oauth account but does not yet exist as 
-// a user in your app, you can use the transfer method to forward that 
-// information.
-
-const userNeedsToBeCreated =
-  signIn.firstFactorVerification.status === "transferable";
-
-if (userNeedsToBeCreated) {
-  const res = await signUp.create({
-    transfer: true,
-  });
-  if (res.status === "complete") {
-    window.Clerk.setActive({
-      session: res.createdSessionId,
-    });
+So for maximum security in your production instances, you need to allowlist your custom redirect URLs via the [Clerk Dashboard](https://dashboard.clerk.com/) or the [Clerk Backend API](/docs/references/backend/redirect-urls/create-redirect-url).
+
+To allowlist a redirect URL via the Clerk Dashboard:
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/social-connections).
+2. In the navigation sidebar, select **User & Authentication > Social Connections**.
+3. Scroll to the **Allowlist for mobile OAuth redirect** section and add your redirect URLs.
+
+## OAuth account transfer flows
```

Comment:
This could all be part of the main OAuth custom flow example. A transfer flow is super common, and should be added to most.

There are some edge cases where customers don't want it, so maybe keeping it separate makes sense. If so, then I would move this up so its under the '[Create the sign-up and sign-in flow](https://clerk.com/docs/pr/1045/custom-flows/oauth-connections#create-the-sign-up-and-sign-in-flow)' and this would really be a part 2 for most customers.

I would change the paragraph to be more like (rough draft):
----
It is common for users with OAuth to use a sign-in button when they need to sign-up, or vice versa. For those cases the `useSignIn` and `useSignUp` hooks have a `transferable` status that you can use to gracefully switch flows programmatically as needed. Most customers will want to add this to their custom flow unless they want to strictly separate sign-ins and sign-ups.